README.md

```diff
@@ -6,27 +6,71 @@ easy-update-metadata-dataset
 SYNOPSIS
 --------
 
-    easy-update-metadata-dataset [--doUpdate] <datasets.csv>
+    easy-update-metadata-dataset [--doUpdate] <input-file> [<complex_values_directory>]
 
 
 DESCRIPTION
 -----------
 
 Batch-updates XML streams of objects in a Fedora Commons repository.
+The command is idempotent, so running it either once or multiple times gives the same result.
 
 
 ARGUMENTS
 ---------
 
-          --doUpdate   Without this argument no changes are made to the repository, the default is a test mode
-                       that logs the intended changes
-      -h, --help       Show help message
-      -v, --version    Show version of this program
-    
-     trailing arguments:
-      input-file (required)   The CSV file (RFC4180) with required changes. The first line must be
-                              'FEDORA_ID,STREAM_ID,XML_TAG,OLD_VALUE,NEW_VALUE', in that order. Additional columns
-                              and empty lines are ignored.
+           --doUpdate   Without this argument no changes are made to the repository, the default is a test mode
+                               that logs the intended changes
+       -h, --help       Show help message
+       -v, --version    Show version of this program
+         
+       trailing arguments:
+           input-file (required)                     The CSV file (RFC4180) with required changes. The first line must
+                                                     be 'FEDORA_ID,STREAM_ID,XML_TAG,OLD_VALUE,NEW_VALUE', in that
+                                                     order. Additional columns and empty lines are ignored.
+           complex_values_directory (not required)   A reference to a directory containing files with the complex
+                                                     values. These files are referenced in the input-file.
+                              
+                              
+EXAMPLES
+--------
+
+###Example: update simple value in EMD
+
+This module was developed to update simple values in the EMD, like the AccessCategory. 
+`easy-update-metadata-dataset --doUpdate changeAccessCategory.csv`
+
+```csv
+FEDORA_ID,      STREAM_ID, XML_TAG,      OLD_VALUE,    NEW_VALUE
+easy-dataset:1, EMD,       accessRights, GROUP_ACCESS, OPEN_ACCESS
```

Comment:
As this should also be changed in the `DC` stream, please add the extra row for that as well. Not only as an example, but also as a reminder to the one who is seeking to use this tool for updating dataset metadata in Fedora.